### DISCUSSION TITLE
DNS_PROBE_FINISHED_NXDOMAIN when running a Flask app from the codespaces

### DISCUSSION THREAD

**Discussion by rhythmd18** [community/community#45992] [Codespaces]
### Select Topic Area

General

### Body

I created a Flask application on the GitHub Codespaces. I am not able to access the webpage by running the app as the webpage does not show up and instead it throws an error: "DNS_PROBE_FINISHED_NXDOMAIN". 
What's the actual issue here, and how do I possibly fix this?

**Comment by jge162** [comment]
Check the firewall settings: Make sure that the firewall is not blocking the connection to the web server.

Try to flush your DNS cache by running the command `ipconfig /flushdns` on Windows
